docs/user-research-prompts.md

```diff
@@ -0,0 +1,328 @@
+* For groups of users, such as the group attending a Suites Guild session, use
+  a *focus group* (possibly including card sorting exercises) which revolves
+  around a volunteer user working on a specific pre-arranged suite
+  problem (e.g. relating to configuration, scheduling, runtime or design of)
+  taken from their day-to-day work on their laptop, as shared on the screens,
+  & describing some brief background & how they use C&R to go about solving it.
+* As well as the above, invite & encourage *random pieces of user feedback*,
+  via the usual channels but also by setting up & advertisting a form for that
+  purpose only. Users may think of something important to convey during their
+  day-to-day work, & this would allow them to note it down easily at the
+  time, where it otherwise may be forgotten.
+
+
+### 7. Should we consider historical user feedback, & if so how?
+
+#### Key considerations:
+
+* From user support over the years, we have heard & observed a lot,
+  including information that could be very useful.
+* However, the context in which it was said is really important. For example,
+  a user might have conveyed a more negative opinion than they really hold
+  at a stressful instance (a deadline or when encourntering a bug) or have
+  made a comment about an aspect of C &/ R that has since been
+  modified so that it is no longer applicable.
+
+#### My proposals:
+
+* Consider signifcant past feedback only where it is evidenced visibly,
+  (typically in writing) & not just recalled from someone's memory, & only
+  where there is enough background evidenced to provide context to record.
+* To uncover & expose useful feedback conveyed in the past in this way,
+  we should systematically trawl (i.e. skim read) through our written
+  communications channels, including the Google Groups & Metomi team inboxes
+  & relevant MO Yammer groups, & copy & paste into the UR records any relevant
+  feedback *alongside brief written metadata notes about the context
+  it was made in* (the date conveyed on, the channel, the user, the
+  immediate circumstances, etc).
+* Make reasonable effort to check whether any historical comments are still
+  (likely) valid, e.g. by looking at the codebase to see if the utility/aspect
+  in question has not since been changed in a relevant way.
+
+
+### 8. Where do we record all user research?
+
+#### Key considerations:
+
+* Information is useless unless it can be *seen*.
+* It is much easier to organise & analyse information if it is all *kept in one
+  location*.
+* It would be helpful if we could all freely add to & edit documents holding
+  UR, with knowledge that we won't undo or block other people's amendments.
+* It is *not wise to disclose the store of UR with any users* we are collecting
+  input from, as they will be influenced & biased by other people's comments.
+
+#### My proposals:
+
+* Use a contained location (repo or directory) in GitHub to record all UR as
+  it is accumulated.
```

Comment:
But once it is all accumulated and we are not collecting any data, can we maybe move it to an issue, maybe in cylc-admin?

Reply:
We will probably need to sift through it to pick out concrete, self-contained & realisable requests for improvements & new features etc., but it would definitely be a good idea to move such useful feedback as relevant to the Issue tracker, which is our main way of keeping up-to-date on that. :+1: